src/language-graphql/printer-graphql.js

```diff
@@ -128,7 +128,7 @@ function genericPrint(path, options, print) {
         return [
           '"""',
           hardline,
-          join(hardline, node.value.replace(/"""/g, "\\$&").split("\n")),
```

Comment:
I think it's a bug in parser, check the `StringValue.value` the leading space are removed, or is there a spec says it should be removed?

**Prettier 2.6.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAdNHVQAS4MICuAzjBALZwBOWu2hJ5VW6LasAngA5zYAKVRaNmA1sUAIYUk2AMoxKASygBzLAF9mUVixy16pCtV11iBplsxsYXHv0qCcI3RKmz5S1VDUgANCAicMArQRMig4pSUEADuvBEIoSjiADbR4uyhfgBGlOJgANZwMDKScAAySnDIAGYpRHDZuQVFMpx5HsjyBA0gcGRZcAAmg0Nl4ioE4spwAGIQlGTiMEEqyCDiBKS+IAAWMGTJAOo7CvBEbWBwMgmnCgBup+xrYESZIEr1lDC8ucqLNXUegArIgADxkHmScAAigQIPAAcl6n42vYqGtlLlODsAI7JbacRSwQ4KQYwHbIAAcAAYUVF6ocsWtCXBPncqn4cXD4D8Aol1kQALRQOBDIbbShwLkKSU-Kb-JC1JE9epkBSdSjdPxESEw7lVRWAvwwcRZElkilIABMxtyCmSHjw5AVvSIAFZtsQ4AAVU2JJXIkB3boASSgI1gMjAikCAEFw3J2FDEfU1GogA)
<!-- prettier-ignore -->
```sh
--parser graphql
```

**Input:**
<!-- prettier-ignore -->
```graphql
"""
   Customer
   Customer
"""
type Person {
  name: String
}

"""
    Customer
   Customer
"""
type Person {
  name: String
}
```

**Output:**
<!-- prettier-ignore -->
```graphql
"""
Customer
Customer
"""
type Person {
  name: String
}

"""
 Customer
Customer
"""
type Person {
  name: String
}

```

Reply:
I'm not familiar with graphql, why it's called `StringValue` not `Comment` or something?

Reply:
Ah, maybe I start understanding, it's a `description`, so may be we should only apply logic to the `StringValue ` in `description`?

Reply:
There is spec:
https://spec.graphql.org/October2021/#sec-Descriptions

> GraphQL descriptions are defined using the Markdown syntax (as specified by [CommonMark](https://commonmark.org/)).

Reply:
Added `Markdown` support in GraphQL `Descriptions`. This should resolve the weird use cases you encountered with jumping leading spaces.